ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -89,9 +89,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
         mkdir build
     fi
     cd build
-    cmake -DCMAKE_INSTALL_PREFIX=${NNST_ROOT} \
-    -DINCLUDE_INSTALL_DIR=${NNST_ROOT}/include \
```

Comment:
@wooksong  If you can, could you comment the local test result  with the pr-audit-nnstreamer-ubuntu-apptest.sh to avoid unexpected issue in the real server?

Reply:
@leemgs Sure!